tasks/buildimages.py

```diff
@@ -2,24 +2,32 @@
 
 import os
 
-from invoke import Context, task
+import yaml
+from invoke import Context, Exit, task
 
-from tasks.pipeline import update_circleci_config, update_gitlab_config, update_test_infra_def
+from tasks.libs.ciproviders.circleci import update_circleci_config
+from tasks.libs.ciproviders.gitlab_api import ReferenceTag, update_gitlab_config, update_test_infra_def
+from tasks.libs.common.color import color_message
 
 
 @task(
     help={
-        "image_tag": "tag from build_image with format v<build_id>_<commit_id>",
-        "test_version": "Is a test image or not",
+        "tag": "tag from build_image with format v<build_id>_<commit_id>",
+        "images": "The image(s) to update, comma separated. If empty, updates the DATADOG_AGENT_ buildimages. Can be a incomplete pattern, e.g. 'deb,rpm' will update all deb and rpm images. deb_x64 will update only one image. Use `all` to update all images",
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ Is there an exhaustive list to make it easier to reference possible patterns? Just the `gitlab-ci.yml` variables section?

Reply:
Yes just this. Maybe I could add an option to list possible values.

Reply:
Something to think about - I don't know if I'd have a good idea of _what_ I could provide here that isn't "all" but if we're not usually needing a subset, it may als obe fine

Reply:
Added an option to list values, with `--list-images` on [this commit](https://github.com/DataDog/datadog-agent/pull/30380/commits/aff1a68aefe20cdddc44f711a5c1e851b7549878)